CMakeLists.txt

```diff
@@ -16,6 +16,7 @@ include(cmake/tinyxml2.cmake)
 include(cmake/zlib.cmake)
 
 if(NOT MSVC)
+    cmake_policy(SET CMP0058 NEW)
```

Comment:
Do we use the `BYPRODUCT` anywhere?

Reply:
No idea. It's somewhere in the cpp_rsc cmake code.